crates/ruff_python_formatter/src/expression/string.rs

```diff
@@ -323,14 +323,21 @@ fn preferred_quotes_raw(
                     break true;
                 }
 
-                if chars.peek() == Some(&configured_quote_char) {
-                    // `""` or `''`
-                    chars.next();
+                match chars.peek() {
+                    // We can't turn `r'''\""'''` into `r"""\"""""`, the last previously inner quote
+                    // we shorten the quoted part and turn the last triple quote char into an
+                    // unterminated string start.
```

Comment:
I don't understand this comment. What I understand from the code is that the match now returns `true` for a single quote at the end of the string. What about a triple quoted string that ends with two quotes?
```'''""'''```

Reply:
good point, fixed